pori_python/ipr/ipr.py

```diff
@@ -627,30 +632,77 @@ def get_kb_matches_sections(
 
 
 def get_kb_disease_matches(
-    graphkb_conn: GraphKBConnection, kb_disease_match: str = None, verbose: bool = True
+    graphkb_conn: GraphKBConnection,
+    kb_disease_match: str = None,
```

Comment:
Doesn't assigning None to string create a type error?

Typing would want:
`kb_disease_match: str = "",`
or
`kb_disease_match: Optional[str] = None `

Reply:
Right. Will do ```kb_disease_match: Optional[str] = None ``` as I usually like having the possibility to distinguish between an empty string vs no argument passed.